snuba/writer.py

```diff
@@ -1,12 +1,12 @@
 import logging
 
-from snuba.clickhouse import ALL_COLUMNS, Array
+from snuba.clickhouse import get_all_columns, Array
 
 
 logger = logging.getLogger('snuba.writer')
 
 
-def row_from_processed_event(event, columns=ALL_COLUMNS):
+def row_from_processed_event(event, columns=get_all_columns()):
```

Comment:
Normally we shouldn't do default argument value assignment like this, but in this case it's fine, and I think this is going away in a following review? (If it's not going away soon, this should default to `None` and be set in the function body — but in our specific usage right now, having it evaluate only once is actually a benefit and not a drawback as long as it's never mutated.)

Reply:
it is going away in the PR right above this one